src/components/Header.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Text from "./Text";
+import Title from "./Title";
+import background from "../assets/renature/background.svg";
+
+const Wrapper = styled.div`
+  position: relative;
+  padding: 237px 0 123px;
+
+  background-color: ${(props) => props.theme.colors.darkPrimary};
+  background-image: url(${background});
+  background-size: cover;
+  color: ${(props) => props.theme.colors.white};
+`;
+
+/* TODO: Make responsive */
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+
+  margin: 0 auto;
+  max-width: 1000px;
+`;
+
+const Cell = styled.div`
+  flex: 1 1 auto;
+`;
+
+/* TODO: Replace placeholder */
+const Badge = styled.div`
+  background-color: ${(props) => props.theme.colors.lightNeutral};
+  width: 350px;
+  height: 350px;
+  margin-right: 100px;
+`;
+
+const Ribbon = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  max-width: 220px;
+
+  color: ${(props) => props.theme.colors.white};
+`;
+
+const Triangle = styled.div`
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-style: solid;
+  border-width: 292px 283px 0 0;
+  border-color: ${(props) => props.theme.colors.primary} transparent transparent
+    transparent;
+
+  transform: skew(0deg);
+`;
+
+const RibbonText = styled(Text)`
+  position: relative;
+
+  padding: 43px 0 0 53px;
+`;
+
+const Nav = styled.ul`
+  margin-top: 24px;
+
+  display: flex;
+  flex-direction: row;
+
+  > * + * {
+    margin-left: 24px;
+  }
+
+  a,
+  a:link {
+    color: ${(props) => props.theme.colors.white};
+  }
+`;
+
+const Header = ({ content }) => (
+  <Wrapper>
+    <Row>
+      <Cell>
+        <Badge>Badge Placeholder</Badge>
+      </Cell>
+      <Cell>
+        <Title size="xlarge" as="h1">
+          {content.title}
+        </Title>
+        <Text size="xlarge" as="h2">
+          {content.description}
+        </Text>
+        <Ribbon>
+          <Triangle aria-hidden="true" />
+          <RibbonText size="xxsmall">
+            Another OSS
+            <br /> project by
+            <br /> Formidable
+          </RibbonText>
+        </Ribbon>
+        <Nav>
+          {content.nav.map((navItem) => {
+            // TODO: Create a smart Link component
+            return (
+              <Text size="large" as="li">
+                <a href={navItem.href}>{navItem.label}</a>
```

Comment:
:scream: routing :scream: Should `Link` from `react-router-dom` be used here? 

(I may not understand the big picture just yet...)

Reply:
Not sure if I fully understand the big picture either but this might help clear that up. Are the landers intended to be essentially standalone or still closely tied to the docs system/architecture?

Reply:
Good question! These components will be closely tied to the `react-static` apps that consume them. However, I think `Link` and `NavLink` are the only elements they render that are specific to that architecture. We could do something like letting these templates take a prop to define how links should be rendered.

Reply:
Ok, so... the config would update to something like this: 

```js
const config = { 
   linkComponent: <Link />,
   header: { 
      ...
```

and then render it like this:
 
```js 
const LinkFromConfig = linkComponent;
return ( 
  <LinkFromConfig href={navItem.href}>{navItem.label}</LinkFromConfig>
   ...
```

is that along the lines of what you were thinking?

Could we add styles to it too? e.g. `const StyledLinkFromConfig = styled(linkComponent)`

Reply:
Yeah, I think that works. I think for most of the landers the link component will render either a `react-router-dom` `Link` or a noopener `a` depending on whether it's an external href. For internal links, it will also alter the href to account for the basePath like it does here: https://github.com/FormidableLabs/spectacle/blob/main/docs/src/components/sidebar.js#L47

This will be common enough that it might make sense to build in as default behavior, but I'm on the fence about that.

Reply:
Yep, yep that makes sense. Hmm... would this be a good use case for a higher order component?  I'm just not sure how to account for landers that don't want that default behavior. Maybe there could be a prop to opt out of it? 

At any rate, I will follow the approach above for now, and explore alternate approaches in a separate PR.